airflow/providers/ssh/hooks/ssh.py

```diff
@@ -494,11 +494,13 @@ def exec_ssh_client_command(
                 if recv.recv_ready():
                     line = stdout.channel.recv(len(recv.in_buffer))
                     agg_stdout += line
-                    self.log.info(line.decode("utf-8", "replace").strip("\n"))
+                    for line in line.decode("utf-8", "replace").strip("\n").splitlines():
+                        self.log.info(line)
```

Comment:
```suggestion
                        self.log.info("%s", line)
```

I think this is better for custom formatters. The default formatter just prepend things so there’s no differences, but having the variable separate with the format can have a difference in e.g. secret masking.

Reply:
Secrets are already masked in logs. Also this is not a part of this PR